decidim-admin/app/commands/decidim/admin/create_attachment.rb

```diff
@@ -28,7 +28,7 @@ def call
           @attachment.save!
           broadcast(:ok)
         else
-          if @attachment.errors.has_key? :file
+          if @attachment.errors.key? :file
```

Comment:
I actually like `has_key?` better here. This cop allows configuring it for `is_`, `has_` or `have_` methods, so I could whitelist `has_` methods if you agree.